bats/tests/preferences/verify-paths.bats

```diff
@@ -0,0 +1,99 @@
+# Test case 30
+
+load '../helpers/load'
+# Ensure subshells don't inherit a path that includes ~/.rd/bin
+export PATH=$(echo "$PATH" | tr ':' '\n' | grep -v /.rd/bin | tr '\n' ':')
+
+setup() {
+    if is_windows; then
+        skip "test not applicable on Windows"
+    fi
+}
+
+teardown_file() {
+    load '../helpers/load'
+    run rdctl shutdown
+    assert_nothing
+}
+
```

Comment:
The `teardown_file()` function is provided by `load.bash` and should be deleted here.

Reply:
ok